gdcdictionary/schemas/follow_up.yaml

```diff
@@ -956,9 +956,9 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
-  comorbidity:
+  comorbidities:
```

Comment:
question: are we formally deprecating 'comorbidity' or just kind of deleting it? if we are trying to formally deprecate, do we need to add it to the `deprecated` list instead?

Reply:
added to deprecated properties, kept original entry.